go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	github.com/satori/go.uuid v1.2.0
 	github.com/shirou/gopsutil v0.0.0-20190731134726-d80c43f9c984
 	github.com/sirupsen/logrus v1.4.0
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/testify v1.4.0
```

Comment:
no need for 1.4. if it does not crash any other machine, or breaks compatibility, it is fine.

Reply:
Left it to v1.4.0.